packages/tx/src/message-composers/complex/swap.ts

```diff
@@ -244,3 +244,63 @@ export async function getSwapMessages({
 
   throw new Error(`Unsupported quote type ${quoteType}`);
 }
+
+export function makeSkipIbcHookSwapMemo({
```

Comment:
Giga nit: are IBC hooks exclusive to Skip? Is that why it's in the name?

Reply:
No, we could have different types of hooks, but since this is a Skip smart contract, we need to provide specific parameters that are compatible only with this contract hence why “Skip” is included in the name